packages/@lwc/node-reactions/jest.config.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const BASE_CONFIG = require('../../../scripts/jest/base.config');
+
+module.exports = {
+    ...BASE_CONFIG,
+
+    displayName: 'lwc-node-reactions',
+
+    roots: ['<rootDir>/src'],
+
+    // Override global threshold for the package. As we increase the test coverage we should increase
+    // the threshold as well.
```

Comment:
Is this just a copy/paste comment or is the coverage for this package actually low?

Reply:
It is from Caridy's initial PR for node-reactions https://github.com/salesforce/lwc/pull/1173/files#diff-9a5b37344286d058fc3a43335eeed323. Updated to just use the settings from the base repo.